gradle/libs.versions.toml

```diff
@@ -46,6 +48,8 @@ jaxb-runtime = { module = "org.glassfish.jaxb:jaxb-runtime", version.ref = "glas
 websocket = { module = "javax.websocket:javax.websocket-api", version.ref = "websocket" }
 nimbus = { module = "com.nimbusds:nimbus-jose-jwt", version.ref = "nimbus"}
 jwt = { module = "com.auth0:java-jwt", version.ref = "jwt"}
+service-annotations = { module = "mil.army.usace.hec:service-annotations", version.ref = "service-annotations"}
```

Comment:
I actually meant the plain java implementation of ServiceLoader.

Not that the the lookup library is better or not useful. Mostly just trying to avoid dependencies on the HEC artifact repo for OpenDCS given power issues.

Reply:
Looks like google provides a replacement for service-annotations processor. The named services provided by HEC's (extracted from NetBeans) is pretty nice, but I don't need priority for this and the name can be provided via and interface method. Not too much extra maintenance burden.

Reply:
I should probably go do that in the main codebase. I've been maintaining the service files manually.